Tests/test_imageqt.py

```diff
@@ -38,7 +41,13 @@ def checkrgb(r, g, b):
     checkrgb(0, 0, 255)
 
 
-@pytest.mark.skipif(not ImageQt.qt_is_installed, reason="Qt bindings are not installed")
 def test_image():
     for mode in ("1", "RGB", "RGBA", "L", "P"):
         ImageQt.ImageQt(hopper(mode))
+
+
+def test_closed_file():
+    with pytest.warns(None) as record:
+        ImageQt.ImageQt("Tests/images/hopper.gif")
+
+    assert not record
```

Comment:
Do you know why this is missing coverage? We should have some Qt at least on some CI?

Reply:
It looks like Codecov only received AppVeyor coverage for this PR: https://codecov.io/gh/python-pillow/Pillow/commit/254973f8a035275c342e408e6ec883e5b2d2ee47/build

There are some errors reported:

https://github.com/python-pillow/Pillow/pull/5260/checks?check_run_id=1933751967#step:8:21
```
==> GitHub Actions detected.
->  Issue detecting commit SHA. Please run actions/checkout with fetch-depth > 1 or set to 0
    project root: .
    Yaml found at: codecov.yml
```

https://github.com/python-pillow/Pillow/pull/5260/checks?check_run_id=1933751943#step:15:763
```
->  Pinging Codecov
https://codecov.io/upload/v4?package=bash-20210129-7c25fce&token=secret&branch=imageqt_exclusive_fp&commit=d5f5ae68a8f758e53188903a01f43f693bff3220&build=580803277&build_url=http%3A%2F%2Fgithub.com%2Fpython-pillow%2FPillow%2Factions%2Fruns%2F580803277&name=ubuntu-latest%20Python%203.9&tag=&slug=python-pillow%2FPillow&service=github-actions&flags=GHA_Ubuntu&pr=5260&job=Test&cmd_args=F
{'detail': ErrorDetail(string='Actions workflow run is stale', code='not_found')}
404
```

Reply:
Hmm. I tried rebasing in hopes of getting more reports, but I only got one more.

Reply:
Well we get some coverage here now: https://codecov.io/gh/python-pillow/Pillow/src/dc0db43d35b081ad5f0f6266be79c0355d284f3b/Tests/test_imageqt.py

and https://codecov.io/gh/python-pillow/Pillow/commit/dc0db43d35b081ad5f0f6266be79c0355d284f3b/build shows a load of GHA builds too. Good enough for this PR, let's keep an eye on Codecov...